src/WebJobs.Script.WebHost/Diagnostics/FunctionsSystemLogsEventSource.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Tracing;
+using System.Reflection;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    [EventSource(Guid = "a7044dd6-c8ef-4980-858c-942d972b6250")]
+    public sealed class FunctionsSystemLogsEventSource : ExtendedEventSource
+    {
+        internal static readonly FunctionsSystemLogsEventSource Instance = new FunctionsSystemLogsEventSource();
+
+        [SuppressMessage("Microsoft.Naming", "CA1709:IdentifiersShouldBeCasedCorrectly", Justification = "MDS columns names are Pascal Cased")]
+        [SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate")]
+        [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational, Version = 4)]
+        public void RaiseFunctionsEventVerbose(string SubscriptionId, string AppName, string FunctionName, string EventName, string Source, string Details, string Summary, string HostVersion, string EventTimestamp, string FunctionInvocationId, string ScriptHostInstanceId)
```

Comment:
Can we change the column name to "HostInstanceId"?

Reply:
HostInstanceId - might be confusing as we also have JobHost in webjobs SDK. ScriptHostId is moredescriptive. 
/cc: @paulbatum

Reply:
I don't see the confusion. For a log line there is only a single host (ScriptHost "is a" JobHost) so they have the same instance ID. If you notice, all the other log points are just using "Host" (e.g.  "Starting Host (HostId={hostId}"), so I'm just suggesting you be inline with that, and not leak out this "ScriptHost" terminology everywhere.

Reply:
Yup. Changing it to just InstanceId